test/drivers/test_suites/syscall_exit_suite/accept4_x.cpp

```diff
@@ -221,7 +207,7 @@ TEST(SyscallExit, accept4X_UNIX)
 	evt_test->assert_numeric_param(4, (uint32_t)0);
 
 	/* Parameter 5: queuemax (type: PT_UINT32) */
-	evt_test->assert_numeric_param(5, (uint32_t)unix_max_queue_len);
+	evt_test->assert_numeric_param(5, (uint32_t)QUEUE_LENGTH);
```

Comment:
the mystery is solved, @hbrueckner I don't know if you remember this, but the reason is that we were extracting the `QUEUE_LEN` from the wrong file descriptor kernel side :face_exhaling:

Reply:
@Andreagit97 cool... that's nice ;-) Thanks.